networking/ovn_kubernetes_network_provider/ovn-k-network-policy.adoc

```diff
@@ -0,0 +1,30 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="ovn-k-network-policy"]
+= OVN-Kubernetes network policy
+include::_attributes/common-attributes.adoc[]
+:context: ovn-k-network-policy
+
+toc::[]
+
+include::snippets/technology-preview.adoc[]
+
+OVN-Kubernetes network policy uses a 3 tiered Access Control List (ACLs) to evaluate and apply network policy. Network policies are evaluate in descending order from tier 1 to tier 3.
```

Comment:
So `NetworkPolicy` and `AdminNetworkPolicy` are two different CRDs, features and so giving them the wider umbrella of them all being "NetworkPolicy" makes it confusing.

I would change this to something along the lines of, Kubernetes offers two features that users can use to enforce network security. One of them is the well-known NetworkPolicy API which has been around for a long time. This was designed for application developers and namespace tenants and detailed docs on that can be found <link-to-ovnk-netpol-docs> and the new feature called `AdminNetworkPolicy` allows cluster administrators to enforce non-overridable policies that take precedence over NetworkPolicies.

Something like that is better,,,? Then you can talk how "How OVNKubernetes implements these Kubernetes native features using tiered ACLs -> link to OVN's ACL docs so that those who don't know what that is can read upon that. This feels odd to drop the ball on acl, tier etc without setting some stage on the introduction paragraph...? Start with Kubernetes, spend a few seconds on what these features do generally and then go into OVNK details of ACLs and tiers